src/components/shared/Snackbar/index.tsx

```diff
@@ -1,9 +1,17 @@
 import * as React from 'react';
 
-import { Text, TextStyle, ViewStyle } from 'react-native';
+import { Text, TextStyle, ViewStyle, Dimensions } from 'react-native';
```

Comment:
'Dimensions' must be in the first position to keep the import sorting rule.
It makes the CircleCI test failed. 
If your code didn't pass the CircleCI tests and you want to what is the reason just click the 'Details' button. 
Then you can find the reason. 
![image](https://user-images.githubusercontent.com/17980230/71670511-654a8580-2db3-11ea-872e-9f04b6a6b791.png)

Reply:
I'm learning a lot thanks to you!